### PR TITLE
Feature/3, 10 locate me & footer

### PR DESCRIPTION
**Pull request by cechae**
1. Add "locate me" button in the landing page
2. Add disclaimers on the footer
3. Add "material-ui/core"

### CODE REVIEW

File: src/components/Navbar/LocateMeButton/LocateMeButton.js [hunk 1]
```diff
@@ -1,5 +1,40 @@
...
+  // const fetchLocationName = async (lat,lng) => {
+  // };
+
+  function onSuccess(pos) {
+    console.log("you running..")
+    let crd = pos.coords;
+    console.log(`More or less ${crd.accuracy} meters.`);
+    let lat = crd.latitude.toString(); 
+    let lng = crd.longitude.toString(); 
+    //var coordinates = [lat, lng];
```
Comment (theborgh) [inline src/components/Navbar/LocateMeButton/LocateMeButton.js]: this was added on App-level

**Review by theborgh — CHANGES_REQUESTED** [review]

File: src/components/Navbar/LocateMeButton/LocateMeButton.js (hunk 1 above)
Comment (cechae) [reply src/components/Navbar/LocateMeButton/LocateMeButton.js]: removed!

**Comment by theborgh** [comment]
Looks good!

**Review by theborgh — APPROVED** [review]

**Review by nhi-ngo — APPROVED** [review]
Looks good to me too. Thank you.

**Pull request merged**